services/app-web/src/components/SubmissionSummarySection/ContractDetailsSummarySection/ContractDetailsSummarySection.tsx

```diff
@@ -206,22 +167,24 @@ export const ContractDetailsSummarySection = ({
                         }
                     />
                 </DoubleColumnGrid>
-                {submission.contractType === 'AMENDMENT' && (
+                {isContractWithProvisions(submission) && (
                     <DoubleColumnGrid>
                         <DataDetail
                             id="modifiedProvisions"
-                            label="This contract action includes new or modified provisions related to the following"
+                            label={isBaseContract(submission)? "This contract action includes provisions related to the following" : "This contract action includes new or modified provisions related to the following"}
                             explainMissingData={
-                                amendmentProvisionsUnanswered && !isSubmitted
+                                provisionsAreInvalid && !isSubmitted(submission)
                             }
                         >
-                            {amendmentProvisionsUnanswered ? null : (
+                            {provisionsAreInvalid ? null : (
                                 <DataDetailCheckboxList
                                     list={modifiedProvisions}
                                     dict={
-                                        isCHIPOnly
-                                            ? CHIPModifiedProvisionsRecord
-                                            : ModifiedProvisionsRecord
+                                        isCHIPOnly(submission)
```

Comment:
Would a helper function that gets the Record for a given submission be used multiple places?

Reply:
Ha, I wrote one ( `getProvisionDictionary` ) looks like I forgot to use it. Will fix this.